### PR TITLE
Brand Usecase

### PR DESCRIPTION
**Pull request by NirmitiJain**
Todo: Update versions

### CODE REVIEW

**Review by narayana-plivo — CHANGES_REQUESTED** [review]
please check the api contract.

File: lib/plivo/resources/brand.rb [hunk 1]
```diff
@@ -76,6 +76,15 @@ def create(options=nil)
...
         perform_create(options)
       end
+
+        ##
+        # Get BrandUsecase
+        # @param [String] brand_id
+      def get_usecases(brand_id)
+        valid_param?(:brand_id, brand_id, [String, Symbol], true)
+        identifier = brand_id + '/usecases'
+        perform_action_with_identifier(identifier, 'GET', nil)
```
Comment (narayana-plivo) [inline lib/plivo/resources/brand.rb:86]: what is the type of output here? Object or Json. I can see there is new fucntion added to_json_brand_usecase.

**Review by narayana-plivo — COMMENTED** [review]
to_json_brand_usecase do we need this?

File: lib/plivo/resources/brand.rb (hunk 1 above)
Comment (NirmitiJain) [reply lib/plivo/resources/brand.rb:86]: It returns an object here, but in UT we need to compare jsons. Hence added the function in spec/resource_brand_spec.rb

Comment (narayana-plivo) [reply lib/plivo/resources/brand.rb:86]: other brand functions also returning object? why different behaviour?

Comment (NirmitiJain) [reply lib/plivo/resources/brand.rb:86]: yes, other functions are also returning response object

**Review by narayana-plivo — APPROVED** [review]
LGTM

**Pull request merged**